app/views/adjustments/index.html.erb

```diff
@@ -1,19 +1,22 @@
 <% content_for :title, "#{current_organization.name} - Inventory - Adjustments" %>
 
-<div class="header-action">
-  <%= link_to "New Adjustment", new_adjustment_path, class: "button float-right" %>
-</div>
-
-<h1>Adjustments</h1>
-
+<% content_for :sidebar do %>
 <section id="filters">
+  <h2>Filters</h2>
   <%= form_tag(adjustments_path, method: :get) do |f| %>
-    <%= label_tag "Show only this Adjustments at this Storage Location" %>
+    <%= label_tag "by Storage Location" %>
     <%= collection_select(:filters, :at_location, @storage_locations || {}, :id, :name, { include_blank: true, selected: @selected_location }) %>
```

Comment:
need to add a .uniq or something to the @storage_locations to get rid of duplicates :)

Reply:
ah crap I thought it was handling that in the controller. ok one min!

Reply:
fixed!

Reply:
Yay! Much better doing it in the controller too :)